santosfamilyfoundation/RuinaMain
# [RUIN-274] Confirm autofill functionality

# Description

Previously the app had autocomplete buttons for weather conditions, data/time, location, driver's license, VIN, and license plate. However not all the autocomplete features appeared necessary or fully functional. Now the app only has the autocomplete buttons for date/time, location, and driver's license.

# Testing (must be done on device)

1. Open the app
2. Create a new report
3. Add a vehicle
4. Open the Crash/Road Form
5. Check the date/time button works
6. Check the location buttons work
7. Check weather conditions only have a multiselect dropdown
8. Open the Vehicle Form
9. Check VIN only has a text box
10. Check license plate only has a text box
11. Open the Driver Form
12. Check driver's license button works
